pom.xml

```diff
@@ -96,6 +96,11 @@ THE SOFTWARE.
         <artifactId>access-modifier-annotation</artifactId>
         <version>1.34</version>
       </dependency>
+      <dependency>
+        <groupId>org.bouncycastle</groupId>
+        <artifactId>bcpkix-jdk18on</artifactId>
+        <version>1.78.1</version>
+      </dependency>
```

Comment:
Does not need to be in `dependencyManagement`; you can just specify the `version` directly in `dependencies`.